content/docs/esc/integrations/_index.md

```diff
@@ -10,6 +10,7 @@ menu:
     weight: 6
 aliases:
     - /docs/esc/providers/
+    - /docs/pulumi-cloud/esc/providers/ 
```

Comment:
This should be this instead (and was indeed surfaced [in the broken-links report in #docs-ops](https://pulumi.slack.com/archives/C04LH6RQZHT/p1726419345123749) before the release yesterday):

```suggestion
    - /docs/esc/integrations/dynamic-login-credentials/
```